schemas/apps.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/aiidalab/aiidalab-registry/v1/schemas/apps.schema.json",
+    "$ref": "#/definitions/Welcome",
+    "definitions": {
+        "Welcome": {
+            "type": "object",
+            "propertyNames": {
+              "pattern": "^[A-Za-z][A-Za-z0-9-_]*$"
+            },
+            "additionalProperties": false,
+            "patternProperties": {
+                ".*": {"$ref": "#/definitions/App"}
+            },
+            "title": "Schema for the AiiDAlab apps registry."
+        },
+        "App": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "git_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "qt-uri-protocols": [
+                        "https"
+                    ],
+                    "qt-uri-extensions": [
+                        ".git"
+                    ]
+                },
+                "meta_url": {
+                    "type": "string",
+                    "format": "uri",
+                    "qt-uri-protocols": [
+                        "https"
+                    ],
+                    "qt-uri-extensions": [
+                        ".json"
+                    ]
+                },
+                "categories": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                          "classical",
+                          "quantum",
+                          "setup",
+                          "tutorials",
+                          "utilities"
```

Comment:
I was wondering, is it possible to take this info directly from the `categories.py`?